styles/services.scss

```diff
@@ -7,7 +7,20 @@
 @import "~@carbon/ibmdotcom-styles/scss/patterns/sections/ctasection/index";
 @import "~@carbon/ibmdotcom-styles/scss/patterns/sections/leadspace/leadspace";
 
+@import "~@carbon/ibmdotcom-styles/scss/components/link-list/index";
+
 @import "~@carbon/ibmdotcom-styles/scss/components/card-link/card-link";
+
 @import "~@carbon/ibmdotcom-styles/scss/components/layout/layout";
-@import "~@carbon/ibmdotcom-styles/scss/components/link-list/index";
+
 @import "~@carbon/ibmdotcom-styles/scss/components/tableofcontents/index";
+@import "~@carbon/ibmdotcom-styles/scss/patterns/blocks/callout-data/index";
```

Comment:
@diego-codes can you repoint to `/scss/components` for all of these pattern imports? See updated documentation for all of these components, as we moved them over from `/scss/patterns`.

Reply:
I think you meant to mention @Diego-Marcuz 😉

Reply:
Sorry! wrong diego!

cc: @diegomarcuz